generators/app/tasks/appSetup/crashlyticsFeatureFiles.js

```diff
@@ -0,0 +1,51 @@
+function addConfigToAndroidFiles() {
```

Comment:
@davidtito1992 all these configurations was deprecated. There is a new version of rn firebase that is compatible with rn v0.60
check here 
https://invertase.io/oss/react-native-firebase/v6/crashlytics/quick-start
https://invertase.io/oss/react-native-firebase/v6/crashlytics/android-setup
https://invertase.io/oss/react-native-firebase/v6/crashlytics/ios-setup

Reply:
Done